### PR TITLE
Make Telegram group link India friendly

### PR DESCRIPTION
**Pull request by rohanrajpal**
`t.me` links don't open in India but `telegram.me` links open.

### CODE REVIEW

**Comment by sushrut111** [comment]
I do not think t.me does not work in India. It works for me - I am in India.

**Comment by rohanrajpal** [comment]
image

The same goes for all of my friends in Delhi.

**Comment by sushrut111** [comment]
Try appending https before the url

File: README.md [hunk 1]
```diff
@@ -76,4 +76,4 @@ https://user-images.githubusercontent.com/11269256/118931418-9bff4c80-b964-11eb-
 4. I work at Microsoft India and this project has nothing to do with my employer and I have worked on this outside of my commitment to my employer.
 
 ## Support
-1. Should you need any help or face troubles using this extension, reach out to me via this telegram channel: [https://t.me/cowinbooking](https://t.me/cowinbooking)
+1. Should you need any help or face troubles using this extension, reach out to me via this telegram channel: [https://telegram.me/cowinbooking](https://telegram.me/cowinbooking)
```
Comment (sushrut111) [inline README.md:79]:
```suggestion
1. Should you need any help or face troubles using this extension, reach out to me via this telegram channel: 
    - [https://telegram.me/cowinbooking](https://telegram.me/cowinbooking)
    - [https://t.me/cowinbooking](https://t.me/cowinbooking)
```

Comment (sushrut111) [reply README.md:79]: Let us keep both of them

**Comment by rohanrajpal** [comment]
> Try appending https before the url

image

**Review by sushrut111 — APPROVED** [review]

**Pull request merged**